tests/e2e/modules/clock_spec.js

```diff
@@ -7,8 +7,7 @@ describe("Clock module", () => {
 	});
 
 	const testMatch = (done, element, regex) => {
```

Comment:
general question: cant the tests themself not be written async too? something like

const testMatch = async (elemtn, regex) .... ?

Reply:
currently it looks like a wild mix of callback and async....

Reply:
can you give me a more detailed example how `testMatch` should look like and how it should be called?

Reply:
think I got it, will deliver new commit.